public/styles/styles.css

```diff
@@ -9,6 +9,12 @@ input {
   color: white;
 }
 
+input::-webkit-outer-spin-button,
```

Comment:
you and your spins!!

Reply:
Lmao I wish. So when you create an HTML input type="number", it has tiny little arrows on the side that can't easily be styled. That code disables them, because I made some using buttons anyway.